EXAMPLE.md

```diff
@@ -0,0 +1,65 @@
+Let's make install spy and generate something we can compile with C and run.
+
+Requires:
+- Emscripten
+- Zig
+- Fresh Python 3.12 possibly less, but Python 3.9 doesn't suit
+- gcc/clang
+
+For Mac OS:
+
+```bash
+brew install zig && brew install emscripten
+```
+
+Build native libraries for spy:
+
+```bash
+(cd spy/libspy && make TARGET= && make TARGET=native && make TARGET=wasi)
+```
```

Comment:
I think that `make -C spy/libspy` should be enough, shouldn't it?
It automatically builds all the targets

Reply:
Nice! I didn't know. Fixed